{"uuid": "1bca4ae8-3bd3-46fc-a3ee-3c844b932fdd", "children": ["4d7d94f0-453d-4844-a91b-7980a84183d6"], "befores": [{"name": "driver", "status": "broken", "statusDetails": {"message": "AttributeError: 'WebDriver' object has no attribute 'reset'\n", "trace": "  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 38, in run_old_style_hookwrapper\n    res = yield\n          ^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 139, in _multicall\n    teardown.throw(exception)\n    ~~~~~~~~~~~~~~^^^^^^^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\pluggy\\_callers.py\", line 121, in _multicall\n    res = hook_impl.function(*args)\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1196, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"C:\\Users\\pc\\AppData\\Local\\Programs\\Python\\Python313\\Lib\\site-packages\\_pytest\\fixtures.py\", line 923, in call_fixture_func\n    fixture_result = next(generator)\n  File \"C:\\Users\\pc\\Desktop\\Challenge Gaston Ferreyra\\Mobile-test\\conftest.py\", line 38, in driver\n    driver_instance.reset()\n    ^^^^^^^^^^^^^^^^^^^^^\n"}, "start": 1759530353912, "stop": 1759530358186}], "start": 1759530353912, "stop": 1759530358279}